pkg/kubectl/cmd/testing/BUILD

```diff
@@ -10,9 +7,7 @@ go_library(
         "zz_generated.deepcopy.go",
     ],
     importpath = "k8s.io/kubernetes/pkg/kubectl/cmd/testing",
-    visibility = [
-        "//build/visible_to:pkg_kubectl_cmd_testing_CONSUMERS",
-    ],
+    visibility = ["//visibility:public"],
```

Comment:
Please revert this change.
I guess this is happening because you delete and regenerate it?

Reply:
yeah, usually when bazel complains and regenerating doesn't help as a last resort I delete `BUILD` files 😇